core/src/env_platform/utility.bal

```diff
@@ -48,3 +48,39 @@ function extractTreeInformationAsJSONArray(TreeInformation[] treeInfoArray) retu
     }
     return treeInformation;
 }
+
+# The `constructApplicationVersion` function will construct the application version which will be suitable for the database.
```

Comment:
Will have to change this based on the validity of the question about the method name suitable.